frontend/form-builder/src/app/App.tsx

```diff
@@ -1,34 +1,50 @@
 import React from "react";
 import "@digdir/design-system-tokens/brand/digdir/tokens.css";
 import classes from "./App.module.css";
+import { SubHeader } from "../../../main-app/src/components/SubHeader";
 import { Toolbar } from "../components/Toolbar";
 import { FormBoard } from "../components/FormBoard";
 import { SettingsSidebar } from "../components/SettingsSidebar";
-import { Heading } from "@digdir/design-system-react";
-import { useTranslation } from "react-i18next";
 import { DndProvider } from "react-dnd";
 import { HTML5Backend } from "react-dnd-html5-backend";
+import { Heading, Link } from "@digdir/design-system-react";
+import { FloppydiskFillIcon, TasklistSendFillIcon } from "@navikt/aksel-icons";
+import { useTranslation } from "react-i18next";
 
 export const App = (): React.JSX.Element => {
   const { t } = useTranslation();
   return (
     <>
-      <Heading level={1} spacing className={classes.headline}>
-        {t("form_builder")}
-      </Heading>
-      <div className={classes.builderGrid}>
-        <DndProvider backend={HTML5Backend}>
-          <div className={classes.builderSection}>
-            <Toolbar />
-          </div>
+      <SubHeader>
+        <Heading className={classes.subHeaderHeading} level={2} size="xxsmall">
+          Form Builder
```

Comment:
```suggestion
          {t("form_builder")}
```
There is already a text resource for this :)